tests/loggers/test_base.py

```diff
@@ -48,9 +48,9 @@ def test_logger_collection():
     mock1.agg_and_log_metrics.assert_called_once_with({"test": 2.0}, 4)
     mock2.agg_and_log_metrics.assert_called_once_with({"test": 2.0}, 4)
 
-    logger.close()
-    mock1.close.assert_called_once()
-    mock2.close.assert_called_once()
+    logger.finalize()
```

Comment:
this is missing the status: https://github.com/PyTorchLightning/pytorch-lightning/blob/7ca038b83f9d326b03944e8ffb308f22d0ea241e/pytorch_lightning/loggers/base.py#L304-L310

from the trainer, this seems to always be `"success"`: https://github.com/PyTorchLightning/pytorch-lightning/blob/7ca038b83f9d326b03944e8ffb308f22d0ea241e/pytorch_lightning/trainer/trainer.py#L1270

having `finalize` in loggers vs `teardown` everywhere else is another quirk which we could address in a separate issue/PR